### PR TITLE
Tbd 16

### PR DESCRIPTION
**Pull request by cmtakara**
Created seed script so that there is something in the database.  There is currently only 1 recipe, but more can be added, if we need.

### CODE REVIEW

File: Cohort1/tbd/server/server.js [hunk 1]
```diff
@@ -10,14 +10,21 @@ const mongoose = require('mongoose');
 // useNewUrlParser: true,
 // useUnifiedTopology: true
 // fix deprecation warnings
-  mongoose.connect(process.env.MONGO_URI,{
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
+//////////
```
Comment (allaboutmike) [inline Cohort1/tbd/server/server.js]: We have the git history, so there's no need to keep commented out code. We should remove this.

**Review by allaboutmike — CHANGES_REQUESTED** [review]
Please don't commit commented out code to the repo

**Comment by cmtakara** [comment]
Updated server.js to remove commented out code.
That code was moved from server.js to modes/connection.js

**Review by allaboutmike — APPROVED** [review]
Looks good!

**Pull request merged**